### DISCUSSION TITLE
Finne løsning for å markere eksperimentelle komponenter

### DISCUSSION THREAD

**Discussion by piofinn** [fremtind/jokul#2277] [Annet]
Autosuggest er en eksperimentell komponent, og for å vise dette er det lagt til både en advarsel på portalsiden, og en emoji i tittelen. Emojis er ikke helt i henhold til profilen, så vi bør finne en annen måte å vise dette på. Er det eventuelt nok å ha advarsel på portalsiden? børr de sorteres annerledes?

**Comment by lfbergee** [comment]
Ja, emoji er uheldig, siden vi ikke vet hvordan emojien rendres på ulike platformer.

Vil påstå at det er nyttig at ustabile komponenter merkes tydelig, både i hovedmeny/sidemeny og selve visningen.

Et ikon kan være en løsning. Evt samle de i en submeny som React-hooks.

**Comment by Steinop** [comment]
Grubler litt videre på dette 🤔
I dette tilfellet der Autosuggest egentlig er en ekstra feature til input-feltet tenker jeg at dette egentlig hører hjemme under Inputfeltkomponenten. Regner med det er der den vil havne etter den eksperimentelle fasen. Et spørsmål som dukker opp i den sammenhengen er hvor det er mest naturlig for brukerne av Jøkul å finne komponenten og om vi bør ha et søk istedenfor filter i venstremenyen? 

Kunne vi hatt en egen seksjon nederst på den enkelte komponentsiden for nye/eksperimentelle features i beta der det er relevant? 

Alternativt samle alle/nye eksperimentelle komponenter på én side eller legge de i en submeny som nevnt over? Vi bør prøve å unngå å ha for mange eksperimentelle komponenter liggende over lang tid men samtidig fint å vise work in progress.

**Comment by piofinn** [comment]
@Steinop Enig i at vi trenger et bedre søk i menyene etter hvert. Jeg vet likevel ikke helt om organisering vil løse alt her. Tror også at vi trenger en slags markering på selve komponentsiden, hvis man f.eks. følger en lenke direkte inn.

Angående å ha Autosuggest inn i Text Input stemmer det jo godt, men den er puttet i en egen pakke nå fordi den eksperimentelle versjonen bruker et litt tungt tredjepartsbibliotet som vi ikke ønsker å tvinge på alle som bruker de "vanlige" Input-komponentene. En riktig Autosuggest-komponent bør absolutt inn i samme pakke 😊 For andre typer eksperimentelle features kan en løsning slik du skisserer være et bra alternativ, men tror kanskje vi trenger begge deler.

**Comment by Steinop** [comment]
Bra input @piofinn. Da får vi nok se på en "Ole Brum-løsning" der. (Ja, takk begge deler 😅)

**Comment by Stormoen** [comment]
Legger igjen en kommentar siden jeg delte et skjermbilde av dette i teams. Jeg opplevde emoji som en gøyal løsning på noe in progress! Men forstår argumentene for å løse det på en annen måte. Støtter forslaget med et ikon i Fremtindstil (gjerne med humor på samme måte som reagensrøret!) eller litt kjedeligere men kanskje funksjonelt: en tag etter tittel "under arbeid" eller lignende. 

Synes det allerede er mye grupperinger og valg i menysammenheng - så synes det er synd om man skal komplisere det mer med å samle eksperimentelle komponenter.
